modules/common/src/logger/logger.ts

```diff
@@ -88,16 +71,8 @@ const logObject: Function = (
         };
     }
 
-    if (expressRequest) {
-        const httpRequest: StackDriverHttpRequest = {
-            requestMethod: expressRequest.method,
-            requestUrl: `${expressRequest.protocol}://${expressRequest.host}${expressRequest.path}`,
```

Comment:
I know that in the description of this field is this description ut it makes more sense to me as this example:
https://cloud.google.com/logging/docs/structured-logging#apache-config

so just the path and add server IP for the host